tasks/install.yml

```diff
@@ -0,0 +1,80 @@
+---
+
+- name: Download latest wordpress version
+  get_url:
+    url: https://wordpress.org/latest.zip
+    dest: /tmp/wordpress_latest.zip
+
+- name: Unarchive wordpress zip to /var/www
+  unarchive:
+    src: /tmp/wordpress_latest.zip
+    dest: /var/www/
+    owner: www-data
+    group: www-data
+
+- name: Copy theme
+  unarchive:
+    src: "{{ inventory_hostname }}/{{ wordpress_template }}.zip"
+    dest: /var/www/wordpress/wp-content/themes/
+    directory_mode: yes
+    mode: 0755
+    owner: www-data
+    group: www-data
+  when: wordpress_import_template
+
+- name: remove anonymous user account
+  mysql_user:
+    name: ''
+    host: localhost
+    state: absent
+
+- name: Create a new database with name '{{ wordpress_mysql_database_name }}'
+  mysql_db:
+    name: "{{ wordpress_mysql_database_name }}"
+    state: present
+
+- name: Create mariadb user
+  mysql_user:
+    name: "{{ wordpress_mysql_user }}"
+    password: "{{ wordpress_mysql_password }}"
+    priv: '{{ wordpress_mysql_database_name }}.*:ALL,GRANT'
+    state: present
+
+- name: Fetch random salts for WordPress config
+  local_action: command curl https://api.wordpress.org/secret-key/1.1/salt/
```

Comment:
Can't we generate these salts?

Reply:
we could but i really dont want to, mostly because its what is recommended in the wordpress install guide